rustls-platform-verifier/src/lib.rs

```diff
@@ -71,3 +72,12 @@ pub fn tls_config() -> ClientConfig {
 pub fn verifier_for_dbg(root: &[u8]) -> Arc<dyn rustls::client::ServerCertVerifier> {
     Arc::new(Verifier::new_with_fake_root(root))
 }
+
+/// Return a fixed [SystemTime] for certificate validation purposes.
+///
+/// We fix the "now" value used for certificate validation to a fixed point in time at which
+/// we know the test certificates are valid. This must be updated if the test certificates
+/// are regenerated.
+pub fn verification_time() -> SystemTime {
```

Comment:
This should probably be `pub(crate)` and `#[cfg(test)]` too.

Reply:
Oh! I actually meant to add this to `src/tests/mod.rs` but put it in `src/lib.rs` by mistake. Thanks for flagging. I'll move this and adjust visibility (I don't think the `cfg` gate is required if it's under `src/tests`)

Reply:
Fixed.